includes/Checker/Checks.php

```diff
@@ -105,19 +124,25 @@ protected function run_check_with_result( Check $check, Check_Result $result ) {
 	 *
 	 * @since n.e.x.t
 	 *
-	 * @return array List of plugin check class instances implementing the Check interface.
+	 * @return array An array map of check slugs to Check instances.
 	 */
 	public function get_checks() {
-		// TODO: Add checks once implemented.
-		$checks = array();
-
-		/**
-		 * Filters the available plugin check classes.
-		 *
-		 * @since n.e.x.t
-		 *
-		 * @param array $checks List of plugin check class instances implementing the Check interface.
-		 */
-		return apply_filters( 'wp_plugin_check_checks', $checks );
+		if ( ! isset( $this->checks ) ) {
+			// TODO: Add checks once implemented.
+			$checks = array(
+				'i18n-usage' => new Checks\I18n_Usage_Check(),
```

Comment:
Using underscores is a bit of a better convention than using hyphens. For example, underscores are allowed in variable names and in JS object properties, while hyphens are not.